### PR TITLE
fix: recovered our ellipsis friend

### PR DESCRIPTION
**Pull request by GutierrezGo**
References https://smg-au.atlassian.net/browse/DM-3035

## Before

Screenshot 2024-04-30 at 14 55 47

## After

Screenshot 2024-04-30 at 14 58 52

## How to test

Please check the popover variations: with icon, with label, with long label, etc

### CODE REVIEW

File: src/components/filterPatterns/popover/index.tsx [hunk 1]
```diff
@@ -111,12 +111,14 @@ export const PopoverFilter: FC<PopoverFilterProps> = ({
                     alignItems="center"
                   >
                     {Icon ? <Icon h="xs" w="xs" mr="xs" /> : null}
-                    {[
-                      displayValue ? appliedLabel ?? label : label,
-                      displayValue,
-                    ]
-                      .filter(Boolean)
-                      .join(': ')}
+                    <chakra.span overflow="hidden" textOverflow="ellipsis">
```
Comment (GutierrezGo) [inline src/components/filterPatterns/popover/index.tsx:114]: The `textOverflow` property doesn't work with flexbox directly. Adding additional wrapper for the text  can display again our lovely ellipsis

File: src/components/filterPatterns/popover/index.tsx [hunk 2]
```diff
@@ -111,12 +111,14 @@ export const PopoverFilter: FC<PopoverFilterProps> = ({
                     alignItems="center"
```
Comment (talamcol) [inline src/components/filterPatterns/popover/index.tsx:111]: can we remove it on the span above then?

Comment (GutierrezGo) [reply src/components/filterPatterns/popover/index.tsx:111]: Nuuuuu we cannot because there we have the flex which center the icon and text

Comment (talamcol) [reply src/components/filterPatterns/popover/index.tsx:111]: no no i mean just the `overflow="hidden" textOverflow="ellipsis"` part

Comment (GutierrezGo) [reply src/components/filterPatterns/popover/index.tsx:111]: Nuuuuu we cannot. Otherwise the ellipsis is not showed

Comment (GutierrezGo) [reply src/components/filterPatterns/popover/index.tsx:111]: so, I can remove only the textOverflow="ellipsis" the rest need to be there for displaying the ellipsis. Maybe removing textOverflow="ellipsis" on level up will help us to not break it again.

**Review by talamcol — APPROVED** [review]
🙏

**Pull request merged**